docs/tutorials/from-tabular-data-to-binned-data.ipynb

```diff
@@ -0,0 +1,679 @@
+    "The result is 5-D:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e800d4f1-5851-4394-af7c-bcf07d54e7d8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "binned = sc.bin(table, groups=list(groups.values()))\n",
+    "binned"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1dbcbc78-2f57-45b7-94b2-d8b42d8c0b0d",
+   "metadata": {},
+   "source": [
+    "### Exercise 3\n",
+    "\n",
+    "- Group `table` using `sc.bin` but only by strip and wire.\n",
+    "- Plot the result.\n",
+    "- The wire index increases with the cylinder radius, the strip index increases with z (or decreases with scattering angle).\n",
+    "  Explain the plot, e.g., why is intensity decreasing with increasing wire index?\n",
+    "  \n",
+    "#### Solution"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9d8493c5-2c6e-412b-b31c-b85a0a131882",
+   "metadata": {
+    "tags": [
+     "solution"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "binned_strip_wire = sc.bin(table, groups=[groups['strip'], groups['wire']])\n",
+    "print(\n",
+    "    'Neutrons arrive from the \"left\" in the following figure (low wire index).'\n",
+    "    'They are gradually absorbed so the intensity decreases as we reach deeper '\n",
+    "    'voxel layers:'\n",
+    ")\n",
+    "binned_strip_wire.bins.sum().plot()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8aefdeff-0d4f-4930-8162-ecd4a58cd08f",
+   "metadata": {},
+   "source": [
+    "## From event-based metadata to bin-based metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4323a62a-20cd-42a0-b4fc-62a301ae0e99",
+   "metadata": {},
+   "source": [
+    "For each detected neutron our data records the position of the associated voxel.\n",
+    "After the logical grouping above, every bin corresponds to a voxel.\n",
+    "\n",
+    "It can be more practical to store the voxel position for every bin (voxel) instead of for every event.\n",
+    "This can be achieved, e.g., by computing the mean for every bin.\n",
+    "Note that in this case all events in a voxel record the same voxel position so this proceedure is wasteful &mdash; in practice we may prefer loading the voxel positions directly from a file.\n",
+    "\n",
+    "We can also combine the x, y, and z components into a single array of position vectors:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b01f58f1-a3cc-4d90-a503-841aaeb56401",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pos = sc.zeros(sizes=binned.sizes, dtype=sc.DType.vector3, unit='mm')\n",
+    "pos.fields.x = binned.bins.coords['voxel_x'].bins.mean()\n",
+    "pos.fields.y = binned.bins.coords['voxel_y'].bins.mean()\n",
+    "pos.fields.z = binned.bins.coords['voxel_z'].bins.mean()\n",
+    "binned.coords['position'] = pos\n",
+    "binned"
+   ]
+  },
```

Comment:
Use `sc.show(binned)` to contrast it with the previous case. (Might want to remove event voxel coords.)

Reply:
Not possible since it is 5D. Alternatives?

Reply:
Flatten? Could add an explicit note that this is done for illustration purposes?